src/index.ts

```diff
@@ -119,6 +124,18 @@ class LiskMigrator extends Command {
 			description:
 				'Maximum number of blocks to be iterated at once for computation. Defaults to 100000.',
 		}),
+		network: flagsParser.string({
+			char: 'n',
+			required: false,
+			env: 'NETWORK',
+			description: 'Network for the migration.',
```

Comment:
you could have default as well 🤔 we only have testnet and mainnet, so you can pick a default?